packages/parse5/package.json

```diff
@@ -27,7 +27,15 @@
         "serialize"
     ],
     "license": "MIT",
-    "main": "dist/index.js",
+    "main": "dist/cjs/index.js",
```

Comment:
wouldn't we still want the default to be ESM? so the only way to reach any commonjs entrypoints is by reading the package exports

Reply:
Node versions with ESM support will not read the `main` property if `exports` are specified. That means this field is only for non-ESM Node versions, which should use the CJS version.